jetty-security/src/main/java/org/eclipse/jetty/security/authentication/ClientCertAuthenticator.java

```diff
@@ -213,151 +131,4 @@ public void setValidateCerts(boolean validateCerts)
         _validateCerts = validateCerts;
     }
 
-    /**
-     * @return The file name or URL of the trust store location
-     */
-    public String getTrustStore()
```

Comment:
can we really just remove all these public and protected methods of this class?

Reply:
We should at least deprecate them and have noop behaviours

Reply:
I would say. this was not working or not sure how.. and it's 10 so we can :)